web/src/routes/backOffice/employees/components/EmployeeForm.svelte

```diff
@@ -75,6 +106,10 @@
 			min: 0,
 			max: 30,
 		},
+		password: {
+			min: 0,
+			max: 30,
```

Comment:
As documented on the Swagger spec:
```suggestion
			min: 14,
			max: 72,
```